channels/ON-CHAIN.md

```diff
@@ -11,6 +11,26 @@
 	- [`channel_slash`](#channel_slash)
 	- [`channel_settle`](#channel_settlement)
 
+Each party keeps a state tree specific for the channel. It consists of all the
+channel data: accounts, contracts and etc. and has the same structure as the
+on-chain state tree. Off-chain transactions update this channel auxiliary tree.
+It is a responsibility of the parties to keep this it locally. Solo closing
+transactions provide a proof of inclusion for it instead of
```

Comment:
I understand for "solo closing transactions" you mean channel_close_solo and channel_slash. Ok.